src/pages/home/report/ReportActionsListItemRenderer.js

```diff
@@ -55,6 +55,71 @@ function ReportActionsListItemRenderer({
         ReportUtils.isChatThread(report) &&
         !ReportActionsUtils.isTransactionThread(ReportActionsUtils.getParentReportAction(report));
 
+    /**
+     * Create a lightweight ReportAction so as to keep the re-rendering as light as possible by
+     * passing in only the required props.
+     */
+    const action = useMemo(
+        () => ({
+            reportActionID: reportAction.reportActionID,
+            message: reportAction.message,
+            pendingAction: reportAction.pendingAction,
+            actionName: reportAction.actionName,
+            errors: reportAction.errors,
+            originalMessage: reportAction.originalMessage,
+            childCommenterCount: reportAction.childCommenterCount,
+            linkMetadata: reportAction.linkMetadata,
+            childReportID: reportAction.childReportID,
+            childLastVisibleActionCreated: reportAction.childLastVisibleActionCreated,
+            whisperedToAccountIDs: reportAction.whisperedToAccountIDs,
+            error: reportAction.error,
+            created: reportAction.created,
+            actorAccountID: reportAction.actorAccountID,
+            childVisibleActionCount: reportAction.childVisibleActionCount,
+            childOldestFourAccountIDs: reportAction.childOldestFourAccountIDs,
+            childType: reportAction.childType,
+            person: reportAction.person,
```

Comment:
Is this referenced somehere? I can't find it anywhere.

Reply:
Ah that would be for threads, it would have to be used for that little element showing how many replies are in thread, who responded, right?